Automattic/wp-calypso
# Earn: Rename index.js to index.ts

## Proposed Changes

Update Earn > index to TypeScript. In this case, it is just a file name change. 

## Testing Instructions



1) Checkout this branch, go to `http://calypso.localhost:3000/earn/YOURDOMAIN`, and click around different tabs and sections to confirm navigation still works the same as before. You'll want to connect Stripe if you haven't so that you can navigate between sections behind the Connect to Stripe screen. 

2) This is the only file I've changed in the project to update Earn code that didn't require TS-related file changes. As an extra check, please just pull up this changed file in vscode and confirm you see no TS errors.
